assets/pages/monthly-contributions/components/nameForm.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import TextInput from 'components/textInput/textInput';
+
+
+// ----- Component ----- //
+
+function NameForm() {
```

Comment:
In the future, this component can receive a boolean flag to indicate which version we want. It can be a `First / Last name` or `Full name`.